clients/scanoss/src/main/kotlin/model/BooleanSerializer.kt

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Also here, the commit message should probably just say

    scanoss: Improve the `License` data model